src/views/product-landing/components/get-started-card/get-started-card.module.css

```diff
@@ -0,0 +1,9 @@
+.placeholder {
```

Comment:
I've added a _bunch_ of pretty identical placeholder components.

Intent here is mainly to have a clear picture around prop types for UI components, so that the logic in `transform-raw-content-to-prop` lines up nicely. Secondary intent is to nicely tee things up for subsequent PRs which will implement the non-placeholder UI for each of these components.